.github/workflows/e2e-external.yml

```diff
@@ -0,0 +1,55 @@
+name: End-to-end tests for external PRs
+
+on:
+  pull_request_review:
+    types: [submitted]
+
+concurrency:
+  group: ${{ github.event_name }}-${{ github.ref }}
```

Comment:
Here, `github.ref` [means](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_review):
> PR merge branch `refs/pull/:prNumber/merge`